include/rapidjson/writer.h

```diff
@@ -352,7 +352,7 @@ class Writer {
         char* end = internal::dtoa(d, buffer, maxDecimalPlaces_);
         PutReserve(*os_, static_cast<size_t>(end - buffer));
         for (char* p = buffer; p != end; ++p)
-            PutUnsafe(*os_, static_cast<typename TargetEncoding::Ch>(*p));
+            PutUnsafe(*os_, static_cast<typename OutputStream::Ch>(*p));
```

Comment:
This change looks suspicious.  At least, it means rapidjson's coverage is low in Unicode area.  I think my change is right and the rest (in this file, which are using `TargetEncoding::Ch`) are not though.